.gitignore

```diff
@@ -30,3 +30,6 @@ Session.vim
 .netrwhist
 # auto-generated tag files
 tags
+
+# IntelliJ
+.idea
```

Comment:
Nice catch, should add .DS_Store and similar for atom too. Right now, I honk we have a limited set of stuff in "Editor temp files", maybe we could expand it to `.\*` in the future.

Reply:
I usually try to put only what is stricly necessary in the .gitignore file to avoid unexpected git missing file later on.